x-pack/test/functional/apps/saved_objects_management/import_saved_objects_between_versions.ts

```diff
@@ -20,19 +20,15 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const testSubjects = getService('testSubjects');
   const retry = getService('retry');
 
-  // Failing: See https://github.com/elastic/kibana/issues/116058
-  describe.skip('Export import saved objects between versions', function () {
+  describe('Export import saved objects between versions', function () {
     before(async function () {
-      await esArchiver.load('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.load('x-pack/test/functional/es_archives/getting_started/shakespeare');
+      await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
```

Comment:
The test was failing because it was making the assumption that a vanilla Kibana instance has only one saved object, (the advanced settings / config object). However (and TIL) this is no longer the case.

Adapted the test to empty Kibana before running.


(Plus, importing the data indices was apparently useless in this test so I removed them)